ui/tests/playwright/BrokerPropertyPage.test.tsx

```diff
@@ -0,0 +1,11 @@
+import { page } from "../../jest.setup";
+describe("Brokers property page", () => {
+  test("Brokers property page should display table", async () => {
+    await page.goto(
+      "https://console.amq-streams-ui.us-east.containers.appdomain.cloud/kafka/j7W3TRG7SsWCBXHjz2hfrg/nodes/0/configuration"
```

Comment:
Can we look at running the application in minikube or something similar rather than referencing the demo environment? Otherwise, how can these tests verify the contents of this repo and the changes made in PRs, etc?

Reply:
This will not verify if the changes in the PR will break the demo env, onlv checks if the demo env is presently running as expected. I will start this discussion on slack to weigh our options on this